mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/DefaultProcessorContext.java

```diff
@@ -98,4 +101,9 @@ public CodeGeneratorFactory getCodeGeneratorFactory() {
   public EntityFactory getEntityFactory() {
     return entityFactoryRef.get();
   }
+
+  @Override
+  public boolean getLogsEnabled() {
```

Comment:
Standard JavaBeans conventions mandate that this should be named `isLogsEnabled`.